.github/workflows/test_tap.yml

```diff
@@ -0,0 +1,35 @@
+name: Test tap-github
+
+on: [push, pull_request]
+
+jobs:
+  tests:
```

Comment:
@laurentS - This is a very elegant and simple CI flow. Nicely done. 👍 
Any objection to us replicating this upstream and adapting into the SDK cookiecutter template? 

cc @edgarrmondragon

Reply:
By all means, please do! I mentioned that same idea in my initial comment, it'd be great to reuse it as much as possible :)